src/Tizen.NUI/src/public/Animation.cs

```diff
@@ -631,8 +625,6 @@ public int[] StartTime
         /// <summary>
         /// Gets or sets the end time for each property of the animation.
         /// </summary>
-        /// This will be public opened in tizen_5.0 after ACR done. Before ACR, need to be hidden as inhouse API.
-        [EditorBrowsable(EditorBrowsableState.Never)]
         public int[] EndTime
```

Comment:
This is xaml related change. User will not use this property directly. this is for parsing xaml script Animation definition. to change this, another ACR is required. (This will be fixed later by replacing it with method or interal variable)